demo/src/ChannelController.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Slider, Checkbox } from '@material-ui/core';
+
+const MIN_SLIDER_VALUE = 0;
+const MAX_SLIDER_VALUE = 65535;
+const COLORMAP_SLIDER_CHECKBOX_COLOR = [220, 220, 220];
+
+const style = { display: 'flex', position: 'relative' };
+
+function ChannelController({
+  name,
+  isOn,
+  sliderValue,
+  colorValue,
+  colormapOn,
+  channelOptions,
+  handleChange,
+  disableOptions = false
+}) {
+  return (
+    <>
+      <div style={style}>
+        <select
```

Comment:
Was there a reason not to go with `material-ui`? I'm pretty sure you can basically just capitalize the first letter of each of these (after importing) and have it work.  If not, it's probably pretty close.